Order.Management.PR/PaintingReport.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Order.Management
+{
+    class PaintingReport : Order
+    {
+        public int tableWidth = 73;
+        public PaintingReport(string customerName, string customerAddress, string dueDate, List<Shape> shapes)
+        {
+            base.CustomerName = customerName;
+            base.Address = customerAddress;
+            base.DueDate = dueDate;
+            base.OrderedBlocks = shapes;
+        }
+        public override void GenerateReport()
+        {
+            Console.WriteLine("\nYour painting report has been generated: ");
+            Console.WriteLine(base.ToString());
+            generateTable();
+        }
+
+        public void generateTable()
+        {
+            PrintLine();
+            PrintRow("        ", "   Red   ", "  Blue  ", " Yellow ");
+            PrintLine();
+            PrintRow("Square", base.OrderedBlocks[0].NumberOfRedShape.ToString(), base.OrderedBlocks[0].NumberOfBlueShape.ToString(), base.OrderedBlocks[0].NumberOfYellowShape.ToString());
```

Comment:
All OrderedBlocks are the same structure, which is built by Square, Triangle, and Circle. So we could use a strong type for OrderedBlocks, such as
```
internal class OrderedBlocks
        {
            public Square Square { get; set; }
            public Triangle Triangle { get; set; }
            public Circle Circle { get; set; }
        }
```
It will make the code more readable and not error-prone.